tbx/project_styleguide/templates/patterns/pages/team/team_detail.html

```diff
@@ -2,27 +2,54 @@
 {% load wagtailcore_tags wagtailimages_tags static %}
 
 {% block content %}
-    {% include "patterns/molecules/title-filters/title-filters.html" with item=page meta=page.role %}
+    <div class="grid streamfield">
+        <div class="grid__title">
+            {% include "patterns/atoms/motif-heading/motif-heading.html" with heading=page.title classes="motif-heading--one" %}
 
-    {% image page.image format-webp width-1280 %}
+            <div class="team-page-meta">
+                <div class="tags">
+                    {% for tag in page.teams %}
+                        {% include "patterns/atoms/tag/tag.html" with tag_name=tag.name tag_slug=tag.slug tag_link_base="../" %}
+                    {% endfor %}
+                </div>
+                <span class="team-page-meta__role">{{ page.role }}</span>
+            </div>
+        </div>
 
-    {{ page.intro|richtext }}
-    {{ page.biography|richtext }}
+        <div class="image grid__team-image">
+            {% srcset_image page.image format-webp width-{400,650,950} sizes="(max-width: 598px) 400px, (max-width: 1022px) 950px, (min-width: 1023px) 650px" %}
```

Comment:
I was chatting to Andy the other day, and apparently the new wagtail image tags are way more efficient than creating the renditions individually, so I have experimented here and may try and update other images in the build if we have time. I've managed to get it working in the pattern library yaml, too.

Reply:
Ahh that's awesome. Well done on getting it working in the Pattern Library as well!